homeassistant/components/aqara/base.py

```diff
@@ -0,0 +1,278 @@
+"""Aqara Home Assistant Base Device Model."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import json
+import logging
+from types import SimpleNamespace
+from typing import Any
+
+from aqara_iot import AqaraDevice, AqaraDeviceManager, AqaraPoint
+
+from homeassistant.core import CALLBACK_TYPE, HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.dispatcher import (
+    async_dispatcher_connect,
+    async_dispatcher_send,
+)
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from . import HomeAssistantAqaraData
+from .const import (
+    AQARA_HA_SIGNAL_REGISTER_POINT,
+    AQARA_HA_SIGNAL_UPDATE_ENTITY,
+    AQARA_HA_SIGNAL_UPDATE_POINT_VALUE,
+    DOMAIN,
+)
+from .util import remap_value, string_dot_to_underline, string_underline_to_dot
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class IntegerTypeData:
```

Comment:
This class is unused

Reply:
ok. i remove it first, i will add it when i push next PR for more aqara devices.